src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -722,39 +616,6 @@ class spell_dk_mark_of_blood : public SpellScriptLoader
         }
 };
 
-// 207346 - Necrosis
-class spell_dk_necrosis : public SpellScriptLoader
```

Comment:
This is still correct (all involved spells exist), don't remove scripts only because they are unused (remove ones that are outdated/wrong)

Reply:
I removed all spells that either had a none existing spell or none existing effect; to me it's easier to remove the spell and figure out how to do it then figure out the changes and how to modify the existing one to make the new one.

If you prefer I'll keep them take a pause on the other ones and I'll go over them again and keep the ones that are somewhat close.